presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/util/CassandraCqlUtils.java

```diff
@@ -121,4 +129,27 @@ public static Select selectDistinctFrom(CassandraTableHandle tableHandle, List<C
     {
         return from(select(columns).distinct(), tableHandle);
     }
+
+    public static String getWhereClause(List<CassandraPartition> partitions, String clusteringKeyPredicates)
```

Comment:
No need to extract this method because it's simple and only used in `deleteFrom`.

Reply:
OK. Got it.